src/visitor/ExtractFilterExprVisitor.h

```diff
@@ -50,6 +50,7 @@ class ExtractFilterExprVisitor final : public ExprVisitorImpl {
 
     bool canBePushed_{true};
     std::unique_ptr<Expression> remainedExpr_;
+    std::string *innerVar_;  // used by predicate, reduce and list comprehension
```

Comment:
Where to use this variable?

Reply:
sorry, it's unused and should be deleted